src/receipt/service.js

```diff
@@ -0,0 +1,16 @@
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+import { getConfig } from '@edx/frontend-platform';
+
+const { ECOMMERCE_BASE_URL } = getConfig();
```

Comment:
[inform] just fyi, if `getConfig` is called outside the render cycle of a component like it is here after the imports, only the default config values provided by `frontend-platform` will exist. 

To prevent this (in enterprise), we typically call `getConfig` inside the service functions in case we need to config that was added dynamically via `frontend-platform` initialization with `mergeConfig`, for example.

It's fine as is, but only because `ECOMMERCE_BASE_URL` is provided by default; if it were a custom value added via `mergeConfig`, it'd be undefined here.

Reply:
That's important information, thank you! Will adjust the code, just in case